Sources/Rownd/Rownd.swift

```diff
@@ -11,21 +11,28 @@ import UIKit
 import ReSwift
 import WebKit
 import AnyCodable
+import AuthenticationServices
 
 public class Rownd: NSObject {
     private static let inst: Rownd = Rownd()
     public static var config: RowndConfig = RowndConfig.inst
     public static let user = UserPropAccess()
+    private static var appleSignUpCoordinator: AppleSignUpCoordinator? = AppleSignUpCoordinator(inst)
     
     private override init() {}
     
+    public static func requestAppleSignIn() {
```

Comment:
I think it might be preferable to have a function that looks more like: `requestSignIn(with: RowndSignInHint)`

Then, the sign-in hint is an enum:

```swift
enum RowndSignInHint {
  case appleId
}
```

The function body would look something like:

```swift
public static func requestSignIn(with: RowndSignInHint) {
  switch(with) {
  case .appleId:
    appleSignUpCoordinator?.didTapButton()
  default:
    requestSignIn()
  }
}
```

If a customer wants the standard sign-in process, they simply call `requestSignIn()` as before, but if they want to hint that a particular method should be used, they can call `requestSignIn(with: .appleId)` and Rownd should honor that request if possible.

There may be times when we can't honor it (e.g., if Sign in with Apple isn't enabled within the customer's Rownd app), so a future update might add that logic to fallback to the default in such an event.

Reply:
Updated